packages/lodestar/src/api/impl/validator/validator.ts

```diff
@@ -208,6 +211,11 @@ export class ValidatorApi implements IValidatorApi {
       throw Error("No matching attestations found for attestationData");
     }
 
+    const aggregatorIndex = epochCtx.pubkey2index.get(aggregator);
+    if (aggregatorIndex === undefined) {
+      throw Error("Aggregator pubkey not in epochCtx");
```

Comment:
Should this error contain the pubkey that is not found?

Reply:
sure. We will need to change this anyways to return meaningful error messages in api